src/functions/keycloak/list-users/handler.ts

```diff
@@ -27,22 +15,24 @@ const listUsersHandler: APIGatewayProxyHandler = async () => {
 
     const users = await api.getUsers();
```

Comment:
TS thinks this is of type User form the getUsers method return type definition. But becuase you are doig "filteredUsers" I suspect its not. See previous comment in other file, but this should be typed correctly so it's clear why we are filtering users.

Also I would change the naming, I think in this case we would usually call it mappedUsers. 

If it seems I've misunderstood something lets discuss further